src/tools/arakoon_logger.ml

```diff
@@ -71,30 +71,22 @@ let format_message
       ~ts ~seqnum
       ~section ~level ~lines
   =
+
+  let d =
+    let (f : float) = Obj.magic ts in
+    let df = mod_float f 1.0 in
+    let udf = df *. 1_000_000. in
+    int_of_float udf
+  in
+  let tz_diff = "+0000" in
   Printf.sprintf
-    "%s - %s - %i/0 - %s - %i - %s - %s"
+    "%s %6d %s %s - %i/0 - %s - %i - %s - %s"
```

Comment:
can you make the `/0` part of `%i/0` configurable (while we're breaking backwards compatibility anyway...)?